app/helpers/manage_submissions_helper.rb

```diff
@@ -1,4 +1,11 @@
 # frozen_string_literal: true
 
 module ManageSubmissionsHelper
+    def eligible_for_evaluation?(submission)
+      submission.judging_status.in? ["qualified"]
```

Comment:
not sure this is defined anywhere, but I think it should be all three?
```suggestion
      submission.judging_status.in? ["qualified", "selected", "winner"]
```

Reply:
this and the other status are TBD for existing phoenix functionality